crates/swc_ecma_transforms_base/src/perf.rs

```diff
@@ -50,38 +47,26 @@ where
         N: Send + Sync + VisitWith<Self>,
     {
         if nodes.len() >= threshold || option_env!("SWC_FORCE_CONCURRENT") == Some("1") {
-            GLOBALS.with(|globals| {
```

Comment:
process will panic if `GLOBALS` is not `set` before, if it is set we reset the `GLOBALS` looks a little redundant.